src/Dashboard.js

```diff
@@ -130,14 +133,28 @@ export default class Dashboard extends React.Component {
         }
       });
     return (
+      <div>
       <ChartContainer>
         <div className="dv-Dashboard__content">
-          <Sidebar
-            onFilterChange={this.onFilterChange}
-            scaleFilters={scaleFilters}
-            regionFilters={regionFilters}
-            sdgFilters={sdgFilters}
-          />
+          <h1>Blockchain Impact Ledger</h1>
+        </div>
+      </ChartContainer>
+      <ChartContainer>
```

Comment:
I think using two of ChartContainer was a sort of questionable decision, but it made the layout work, with minimal effort.